demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -67,3 +70,66 @@ def test_integration_secrets_negative(mocker):
     assert "Dynamics365ForMarketingEmail" in result.stdout
     assert "Remove or whitelist secrets in order to proceed, then re-commit" in result.stdout
     assert result.stderr == ""
+
+
+def test_integration_secrets_integration_positive(mocker, tmp_path):
+    """
+    Given
+    - FeedAzure integration yml with secrets.
+
+    When
+    - Running secrets validation on it.
+
+    Then
+    - Ensure secrets validation succeed.
+    """
+    integration_with_secrets_path = join(
+        DEMISTO_SDK_PATH, "tests/test_files/content_repo_example/Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.yml"
+    )
+    mocker.patch(
+        "demisto_sdk.__main__.SecretsValidator.get_all_diff_text_files",
+        return_value=[integration_with_secrets_path]
+    )
+    whitelist = {
```

Comment:
use `create_whitelist_secrets_file` function to create `secrets_white_list file`